lib/Runtime/Types/SimpleDictionaryTypeHandler.cpp

```diff
@@ -199,12 +199,11 @@ namespace Js
         {
             return propertyId;
         }
-        JsUtil::CharacterBuffer<WCHAR> propertyStr(propertyKey->GetString(), propertyKey->GetLength());
-        if (BuiltInPropertyRecords::valueOf.Equals(propertyStr))
+        if (BuiltInPropertyRecords::valueOf.Equals(propertyKey))
```

Comment:
should we pass the propertyId here, I believe you have that overloaded, right?

Reply:
At that point, propertyId is useless, see the code above.